### ISSUE TITLE
TypeScript definitions

### ISSUE THREAD

**Issue by high1** [yuzhva/react-leaflet-markercluster#54]
Missing type definitions for TypeScript.

**Comment by StazriN** [comment]
How it looks with type definitions for TypeScript? When will it be supported please? Thanks

**Comment by rap2hpoutre** [comment]
@gwillz Thank you for posting your workaround! Where did you put your code?

**Comment by silviuburceadev** [comment]
I am using react-leaflet-markercluster 3.0.0-rc1 and @types/react-leaflet-markercluster 2.0.0 and I get this in the code:

> 'MarkerClusterGroup' cannot be used as a JSX component.
  Its instance type 'MarkerClusterGroup<any>' is not a valid JSX element.
    Type 'MarkerClusterGroup<any>' is missing the following properties from type 'ElementClass': render, context, setState, forceUpdate, and 3 more.  TS2786

Code is very simple, used the example:
```
<MarkerClusterGroup>
  <Marker position={[49.8397, 24.0297]} />
  <Marker position={[52.2297, 21.0122]} />
  <Marker position={[51.5074, -0.0901]} />
</MarkerClusterGroup>
```

The index.d.ts clearly says that the definitions apply to v2.0, can we have something for 3.0.0?

Here is the definition for React Leaflet's FeatureGroup:
`export declare const FeatureGroup: import("react").ForwardRefExoticComponent<FeatureGroupProps & import("react").RefAttributes<LeafletFeatureGroup<any>>>;`

So I'm guessing that MarkerClusterGroup is something similar?

**Comment by nunos7** [comment]
Hi, what types do you use?
I'm using "@types/react-leaflet-markercluster": "^3.0.0", and I have no way to set the events :(...

**Comment by charlieforward9** [comment]
Types are now included within this source repo since 3e74e938605b1bc590f9b1b7fb6c4dd59adc404e. 

Bump this dependency to the latest version to take advantage of this resolution.
